src/main/java/me/itzg/tsdbcassandra/entities/Tenant.java

```diff
@@ -0,0 +1,16 @@
+package me.itzg.tsdbcassandra.entities;
+
+import lombok.Data;
+import org.springframework.data.cassandra.core.mapping.PrimaryKey;
+import org.springframework.data.cassandra.core.mapping.Table;
+
+@Table("tenants")
+@Data
+public class Tenant {
```

Comment:
why is this class needed?

Reply:
With previous PoC integrations with UMB, I would tend to rate limit the ingesting to a random percentage of the overall data. As a result, it was an unknown what tenants' data actually got sampled in the end. So this "table" is simply to answer the questions: what all tenants can be picked for validating and is a given tenant present in the dataset?